api/turing/api/router_versions_api.go

```diff
@@ -177,6 +180,24 @@ func (c RouterVersionsController) DeployRouterVersion(
 		"version":   int(routerVersion.Version),
 	})
 }
+func (c RouterVersionsController) ListRouterVersionsByEnsembler(
+	_ *http.Request,
+	vars RequestVars,
+	_ interface{},
+) *Response {
+	options := service.RouterVersionByEnsemblerListOptions{}
```

Comment:
Same comment about this struct; since the new API endpoint name is more generic, the name of this struct can also be more generic too :)